lib/core/src/lib/viewer/components/viewer.component.spec.ts

```diff
@@ -115,6 +115,23 @@ describe('ViewerComponent', () => {
 
     });
 
+    describe('originalMimeType', () => {
+
+        it('should set alt attribute to originalMimeType when originalMimeType is provided', () => {
+            component.originalMimeType = 'image/png';
+            fixture.detectChanges();
+            const altAttribute = fixture.nativeElement.querySelector('.adf-viewer__mimeicon').getAttribute('alt');
+            expect(altAttribute).toBe('image/png');
+        });
+
+        it('should set src attribute based on originalMimeType when originalMimeType is provided', () => {
+            component.originalMimeType = 'image';
+            fixture.detectChanges();
+            const srcAttribute = fixture.nativeElement.querySelector('.adf-viewer__mimeicon').getAttribute('src');
```

Comment:
srcAttribute has any type, please set type: 
<img width="677" alt="image" src="https://github.com/Alfresco/alfresco-ng2-components/assets/115619721/60cbc2af-c69f-4621-9e11-1ef1cfdf0816">

Reply:
The type has been set.